source/frontend/src/models/api/AcquisitionFile.ts

```diff
@@ -45,6 +45,14 @@ export interface Api_AcquisitionFilePerson extends Api_ConcurrentVersion, Api_Au
   isDisabled?: boolean;
 }
 
+export interface Api_AcquisitionFileOwnerSolicitor extends Api_ConcurrentVersion, Api_AuditFields {
+  id?: number;
+  acquisitionFileId?: number;
+  personId?: number;
+  person?: Api_Person;
+  isDisabled?: boolean;
```

Comment:
Should these be null instead?

Reply:
yes, will update. still following the old pattern brainlessly